scripts/library/input.js

```diff
@@ -0,0 +1,31 @@
+//@ts-check
+/** @typedef { HTMLInputElement | null }  HTMLInputElements */
+export default class InputHandler {
+  /** <input> 요소의 값이 비었는지 확인
+   * @param {HTMLInputElements} element - inputElement
+   * */
+  static isEmptyValue = element => {
+    if (!(element?.tagName === 'INPUT')) return;
```

Comment:
이건 왜 괄호로 한번 더 감싸져있나요? 👀 
```suggestion
    if (element?.tagName !== 'INPUT') return;
```

이렇게 해도 동일 할 것 같아요!

Reply:
참고로 클린코드 법칙중에 [드모르간 법칙](https://latte1114.tistory.com/523)이란게 있습니다 ㅎㅎ 참고해보시면 좋을 것 같아요.